recipes/vue-vite/README.md

```diff
@@ -0,0 +1,105 @@
+# Asgardeo Auth Vue.js SDK Usage Example (Single Page Application)
+
+This sample is developed to demonstrate the basic usage of the Asgardeo Auth Vue.js SDK.
+
+## Getting Started
+
+### Prerequisites
+- `Node.js` (version 10 or above).
```

Comment:
Node 18 reaches EOL on April, 2025. Therefore no point in mentioning 10 (Vite and PNPM wouldn't even run on node v10). Let's update this to v20 or above.